opennebula_api/models.py

```diff
@@ -60,15 +61,50 @@ def _get_opennebula_client(self, username, password):
 
     def _get_or_create_user(self, email, password):
         try:
-            user_pool = oca.UserPool(self.oneadmin_client)
-            user_pool.info()
+            user_pool = self._get_user_pool()
             opennebula_user = user_pool.get_by_name(email)
             return opennebula_user
         except WrongNameError as wrong_name_err:
             opennebula_user = self.oneadmin_client.call(oca.User.METHODS['allocate'], email,
                                                         password, 'core')
+        except ConnectionRefusedError:
+            print('Could not connect to host: {host} via protocol {protocol}'.format(
+                    host=settings.OPENNEBULA_DOMAIN,
+                    protocol=settings.OPENNEBULA_PROTOCOL)
+                )
             return opennebula_user
```

Comment:
I think we should have a better way of propagating the "ConnectionRefusedError" to the view?